vit-servicing-station-lib/src/db/queries/funds.rs

```diff
@@ -40,43 +45,66 @@ pub async fn query_fund_by_id(id: i32, pool: &DbConnectionPool) -> Result<Fund,
     .map_err(|_e| HandleError::InternalError("Error executing request".to_string()))?
 }
 
-pub async fn query_fund(pool: &DbConnectionPool) -> Result<Fund, HandleError> {
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FundWithNext {
+    #[serde(flatten)]
+    pub fund: Fund,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub next: Option<FundNextInfo>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FundNextInfo {
+    pub id: i32,
+    pub fund_name: String,
+    #[serde(flatten)]
+    pub stage_dates: FundStageDates,
+}
+
+pub async fn query_current_fund(pool: &DbConnectionPool) -> Result<FundWithNext, HandleError> {
     let db_conn = pool.get().map_err(HandleError::DatabaseError)?;
     tokio::task::spawn_blocking(move || {
-        let fund = fund_dsl::funds
+        let funds: Vec<Fund> = fund_dsl::funds
             // TODO: Not sure if sorting by the PK is actually necessary
+            //
+            // this assumes that the next will be the second inserted
```

Comment:
we should make it clear in the interface as well

Reply:
which interface, exactly?

Reply:
the one we use to insert new funds, but not sure we have that currently :)

Reply:
I mean, it's inserted from a csv... I guess I could add a comment in that argument's documentation. At some point I considered validating that the fund start dates were in order, but then I changed my mind.